ansible/roles/apt/tasks/main.yml

```diff
@@ -108,12 +100,12 @@
   when: (apt__enabled|bool and apt__sources_deploy_state == 'present')
 
 - name: Remove diversion of original /etc/apt/sources.list
-  shell: rm -f /etc/apt/sources.list ; dpkg-divert --quiet --local --rename --remove /etc/apt/sources.list
-  args:
-    warn: False
+  dpkg_divert:
+    path: '/etc/apt/sources.list'
+    state: 'absent'
+    delete: True
```

Comment:
The combination of creation and removal of diversion can be done in a single `dpkg_divert` task - just use the `state` parameter conditionally (the `delete` parameter does nothing with `state: present`). Do you think that you could merge this and other similar tasks into a single task each time this happens? We can leave it for later if it becomes too complex.

Reply:
I'll have a look

Reply:
Ok, done